kolibri/core/content/utils/resource_import.py

```diff
@@ -275,6 +276,16 @@ def run_import(self):
         self.transferred_file_size = 0
         self.file_checksums_to_annotate = []
 
+        channel_has_imported_resources = (
+            ContentNode.objects.filter(channel_id=self.channel_id)
+            .filter(available=True)
+            .exists()
+        )
+
+        if not channel_has_imported_resources:
```

Comment:
We should do this after the import has completed, so as the last thing in the run_import function, otherwise the pingback could report that there is nothing imported in the channel.